File: Iniciante/URI 1015.lua
line1 = {}
n = 0
for i in string.gmatch(io.read("*l"), "%S+") do
   n = n + 1
   line1[n] = i
end
x1 = tonumber(line1[1])
y1 = tonumber(line1[2])

line2 = {}
n = 0
for i in string.gmatch(io.read("*l"), "%S+") do
   n = n + 1
   line2[n] = i
end
x2 = tonumber(line2[1])
y2 = tonumber(line2[2])

distancia = math.sqrt(math.pow((x2 - x1), 2.0) + math.pow((y2 - y1), 2.0))
print(string.format('%.4f', distancia))

File: Iniciante/URI 1015.clj
(require '[clojure.string :as str])

(def line1 (str/split (read-line) #" "))
(def x1 (Double/parseDouble (get line1 0)))
(def y1 (Double/parseDouble (get line1 1)))

(def line2 (str/split (read-line) #" "))
(def x2 (Double/parseDouble (get line2 0)))
(def y2 (Double/parseDouble (get line2 1)))

(def distancia (Math/sqrt (+ (Math/pow (- x2 x1) 2.0) (Math/pow (- y2 y1) 2.0))))

(printf "%.4f\n" distancia)